optuna/study.py

```diff
@@ -105,11 +105,35 @@ def trials(self):
 
         The returned trials are ordered by trial number.
 
+        This is a short form of ``self.get_trials(deepcopy=True)``.
+
+        Returns:
+            A list of :class:`~optuna.structs.FrozenTrial` objects.
+        """
+
+        return self.get_trials()
+
+    def get_trials(self, deepcopy=True):
+        # type: (bool) -> List[structs.FrozenTrial]
+        """Return all trials in the study.
+
+        The returned trials are ordered by trial number.
+
+        For ordinal library users, it's recommended to use more handy
```

Comment:
I'm not sure, but it seems to be a typo. I think we can remove it and call them just `library users`.
```suggestion
        For ordinary library users, it's recommended to use more handy
```

Reply:
Thanks. Your suggestion makes sense. I'll remove the adjective.

Reply:
Fixed: https://github.com/optuna/optuna/pull/696/commits/84d4d838afdfa25054b88d6947a146143a4d5798